apps/outlook/src/assets/icons.tsx

```diff
@@ -1,3 +1,5 @@
+import React from "react";
```

Comment:
Huh, is that needed or just a convenience for future? 🤔

Reply:
It helps the JSX-parsing. You get errors on the XML-tags if you remove it. Reason you don't need it for Next.is for example is cause it's polyfilled there